### PR TITLE
scripts: Add --strip to the android.py install command

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**

### CODE REVIEW

File: scripts/android.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def main():
         build_cmd = f'cmake --build {build_dir}'
         common_ci.RunShellCmd(build_cmd)
 
-        install_cmd = f'cmake --install {build_dir} --prefix {cmake_install_dir}'
+        install_cmd = f'cmake --install {build_dir} --prefix {cmake_install_dir} --strip'
```
Comment (juan-lunarg) [inline scripts/android.py:172]: Only strip if building for release.

**Review by juan-lunarg — CHANGES_REQUESTED** [review]
Should we also do this for the validation layers? It would reduce the size of the binaries we release.

**Comment by jeremyg-lunarg** [comment]
> Should we also do this for the validation layers? It would reduce the size of the binaries we release.

Yes I think so

**Comment by juan-lunarg** [comment]
> Yes I think so

Sounds good to me.

**Comment by juan-lunarg** [comment]
I forgot how much memory does this shave off?

**Comment by jeremyg-lunarg** [comment]
> I forgot how much memory does this shave off?

According to the report we got, something like 5mb -> 800k

**Review by juan-lunarg — APPROVED** [review]

**Comment by juan-lunarg** [comment]
Please do a follow up PR in the validation layers and ping me on it.

**Pull request merged**